src/gui/CMakeLists.txt

```diff
@@ -64,6 +55,12 @@ set(client_UI_SRCS
     wizard/welcomepage.ui
 )
 
+if(QUICK_COMPILER)
+    qtquick_compiler_add_resources(client_UI_SRCS ../../resources.qrc ${CMAKE_SOURCE_DIR}/theme.qrc)
```

Comment:
Looks sane. @claucambra What I am failing to understand is, since you have removed all *.qml* files from `client_UI_SRCS`, how are those getting passed to `qtquick compiler`? Also, are *.ui* files supposed to go to `qtquick compiler`?

Reply:
We are adding them straight through the resources.qrc file, which contains the locations of all the files; qtquick_compiler_add_resources and qt5_add_resources add these files to the client_UI_SRCS list